### DISCUSSION TITLE
Misses animation not being the same as stable

### DISCUSSION THREAD

**Discussion by EXtremeExploit** [ppy/osu#20312] [Ideas] [answered]
in osu! stable when you miss a circle it pops into the screen the miss image like already lazer does, but then it goes down a bit, lazer doesn't do this, it maintains the miss in the same position as the circle, not sure if its intentional or not, but it distracts me a lot when seeing a miss and not dissapearing slowly fading away into the void while also going down a bit

**Comment by Theighlin** [comment, marked as answer]
Duplicate of #15206.
